root/components/UserAccountLayout.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import type {Node as ReactNode} from 'react';
```

Comment:
Apparently the line above this one imports all the types too, according to https://flow.org/en/docs/react/types/, so you can reference `React.Node` without this line.

Reply:
Indeed. Removed with https://github.com/metabrainz/musicbrainz-server/pull/718/commits/862762256e00e7266b46284a41c9a4425cf216e8.